scripts/profiler/model_creator.py

```diff
@@ -141,10 +141,9 @@ def transmit_all_events_descriptions(self):
                     for k, v in self.processed_events.registered_events_types.items())
             json_et_string = json.dumps(event_types_dict)
             try:
-                self.out_stream.send_desc(json_et_string.encode())
+                self.stream.send_desc(json_et_string.encode())
             except StreamError as err:
-                self.logger.error("Error: {}. Unable to send data".format(err))
-                sys.exit()
+                self.logger.error("Sending error: {}. Cannot send descriptions.".format(err))
```

Comment:
The same as comment below:
> In model creator we can still receive data and save it to files, even if one or more events were for some reason not forwarded to plot module. That's why I removed closing. Practically this situation should never occur. Should I reinstate closing here?

Reply:
I would say that if we for some reason cannot send data to the plotting module, we should close (or at least stop trying to send more data to the plot module, because we assume no data drops on our transports in profiler).